CMakeLists.txt

```diff
@@ -103,7 +109,80 @@ aws_use_package(aws-c-http)
 aws_use_package(aws-c-auth)
 aws_use_package(aws-checksums)
 
-# TODO - bundling static libraries into single static library composed of all the static libraries
-# to enable IPO/LTO. See also:
```

Comment:
question: now that we have a single static library, do we get this optimization for free? or is there some extra work that needs to be done

Reply:
I don't know we may have more to do but since it's an optimization I'm not going to worry about it right this second.